src/app/series/directives/series-templates.component.ts

```diff
@@ -11,59 +11,52 @@ import {
   styleUrls: ['series-templates.component.css'],
   template: `
     <form *ngIf="series">
-    <header>
-        <h2>Audio Templates</h2>
-        <p>
+      <publish-fancy-field label="Audio Templates">
+        <div class="fancy-hint">
           When adding episodes to your series, you can have different versions of the audio for
           distribution. For example, it is common to have a podcast version of the audio broken
           into segments for ad injection, but also a broadcast version with different breaks for
           local news, different credits, or edits and bleeps to meet FCC permitted language requirements.
           On this page, you can define basic templates for the versions of audio that will be expected for
           each episode in this series, as well as requirements for each of those templates.
-        </p>
-        <hr/>
-      </header>
+        </div>
+        <button *ngIf="!hasVersions()" class="add-version"
+          (click)="addVersion()"><i class="icon-plus"></i>Add a template</button>
+      </publish-fancy-field>
+
       <template ngFor let-v [ngForOf]="series.versionTemplates">
         <div *ngIf="!v.isDestroy" class="version">
-          <publish-fancy-field textinput required [model]="v" name="label" label="Version Label">
-          <div class="fancy-hint">A name for this audio template, such as "Podcast Audio" or "Clean Version"</div>
-            <div class="actions">
-              <button tabindex=-1 class="btn-link" (click)="removeVersion(v)"><i class="icon-cancel"></i>Remove Template</button>
-            </div>
-          </publish-fancy-field>
-
-          <publish-fancy-field class="length" [model]="v" label="Total length in seconds" invalid="lengthAny">
-            <div class="fancy-hint">The minimum and maximum durations in seconds for the sum of all the audio in this template</div>
-            <publish-fancy-field number small inline hideinvalid [model]="v" name="lengthMinimum" label="Minimum">
+          <header>
+            <strong>{{v?.label}}</strong>
+            <i class="icon-cancel" (click)="removeVersion(v)"></i>
+          </header>
+          <section>
+            <publish-fancy-field required textinput [model]="v" name="label" label="Template Label">
+              <div class="fancy-hint">A name for this audio template, such as "Podcast Audio" or "Clean Version"</div>
             </publish-fancy-field>
-            <publish-fancy-field number small inline hideinvalid [model]="v" name="lengthMaximum" label="Maximum">
+
+            <publish-fancy-field class="length" [model]="v" label="Total length" invalid="lengthAny">
+              <div class="fancy-hint">
+                The minimum and maximum durations for all the audio files. Used to check
+                that your episodes have sane lengths, and prevent uploading bad audio.
```

Comment:
I'd vote against "sane" -- sounds like we'd be judging their episode lengths by standards other than what they're setting here. Maybe something like "Used to ensure that each of your episodes is the desired approximate length, and to prevent uploading bad audio."